spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+include SessionsHelper
+
+RSpec.describe UsersController, type: :controller do
+
+  before do
+    Rails.application.load_seed
+  end
+
+  let(:user) { User.find_by(username: 'user1') }
+
+  context "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+  context "GET #show" do
+    it "returns http success" do
+      get :show, params: { id: 1 }
+      expect(response).to have_http_status(:success)
+    end
+  end
+  context "GET #edit" do
+    context "when user tries to access the wrong profile" do
+      it "redirects user to their own profile" do
+        log_in(User.first)
+        get :edit, params: { id: 2 } # "id: 2" is the wrong user
+        expect(response).to redirect_to(User.first)
+      end
+    end
+
+    context "when correct user accesses edit page" do
+      it "returns http success" do
+        log_in(User.first)
+        get :edit, params: { id: 1 }
+        expect(response).to have_http_status(:success)
+      end
+    end
+
+  end
+
+  # context "PATCH #update" do
```

Comment:
Why is it commented out?

Reply:
This was commented out because we wanted to test it but we could not get it to work. Do you have any idea why this test is failing?